src/main/java/org/dependencytrack/policy/cel/compat/VersionDistanceCelScriptBuilder.java

```diff
@@ -0,0 +1,12 @@
+package org.dependencytrack.policy.cel.compat;
+
+import org.dependencytrack.model.PolicyCondition;
+
+public class VersionDistanceCelScriptBuilder implements CelPolicyScriptSourceBuilder {
+    @Override
+    public String apply(PolicyCondition policyCondition) {
+        return """
+                component.version_distance("%s", "%s")
```

Comment:
For keeping it aligned with legacy evaluator, should it be """.formatted(CelPolicyScriptSourceBuilder.escapeQuotes(policyCondition.getOperator()), policyCondition.getvalue()) ?